src/pages/contributing.md

```diff
@@ -1,43 +1,200 @@
-Contributing to the Documentation
-==================================
+# Contributing to the Documentation
 
-Contribution guidelines are currently being written. You may contribute at a later date.
-<!-- 
+This is a non-exhaustive guideline for making contributions to the [NeoForged Documentation][docs] repository. Contributions can be made by forking and cloning the repository and then added via a pull request, or PR, on the [GitHub][docs].
 
-You can make a contribution via a PR on [GitHub].
+You can run the website locally using [npm]. It is recommended to use a Node Version Manager like [nvm] (Mac, Linux) or [nvs] (Windows) to setup and install npm and Node. From there, you can run the following commands:
 
-This documentation is meant to be explanatory. Please explain how to do things, and break it down into reasonable chunks.
-We have a wiki elsewhere that can capture more comprehensive code examples.
+```bash
+nvm use # or nvs use on Windows
+npm install
+npm run start
+```
 
-Our audience is anyone who wants to understand how to build a mod using Forge.
+## An Overview
 
-Please don't try to turn this documentation into a tutorial on Java Development - it is intended for people who understand how a Java class works, and other fundamental structures of Java.
+This documentation is a guide to help a modder understand and implement a given concept from Minecraft or NeoForged.
 
-Style Guide
------------
+This documentation is NOT meant as a tutorial, allowing a modder to copy-paste the examples. If you are looking for a tutorial, there are plenty of videos and pages, which are not linked here, that you can use and follow along with.
 
-:::caution
-Please use **two spaces** to indent, not tabs.
-:::
+This documentation is also NOT meant as documentation for a class. Providing a description of an element is unavoidable when writing a guide; however, if you would like to document a class, you should contribute to [Parchment for Minecraft][parchment] or [NeoForge for NeoForged][neo].
+
+Final, this documentation is NOT meant to explain Java concepts. This documentation is intended for people who already has a solid basis in Java. If a Java concept needs to be explained to better understand the concept (such as JVM Descriptors for Access Transformers), a link should be provided to the original resource. Otherwise, if you are unfamiliar with Java, there are plenty of online resources to learn from:
+
+* [JetBrains Academy][jetbrains]
+* [Codeacademy][codeacademy]
+* [University of Helsinki][helsinki]
+* [Oracle][oracle]
+* [Introduction to Programming using Java by David J. Eck][eck]
+
+## Concepts
+
+Each page should guide a modder on a particular concept. If the concept is too large in scope, the concept should be split into separate sub-concepts, each within its own page. For example, if you are writing  a cookbook, there can be a page for each recipe, rather than a single page containing all the recipes.
```

Comment:
Double-space! (`if you are writing  a cookbook,`)
```suggestion
Each page should guide a modder on a particular concept. If the concept is too large in scope, the concept should be split into separate sub-concepts, each within its own page. For example, if you are writing a cookbook, there can be a page for each recipe, rather than a single page containing all the recipes.
```